docs/security/using-amz-kms.md

```diff
@@ -1,7 +1,5 @@
 # Using the Amazon Key Management Service (AWS KMS)
 
-This feature is **technical preview** quality.
```

Comment:
Should we mention that as of 8.0.30 the feature is considered Generally Available?

Reply:
Added